website/blog/2022-08-08-aluminium.md

```diff
@@ -0,0 +1,368 @@
+Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- [Scala 3] Auto import and completion for extension methods
+- [Scala 3] Convert to Named Parameters code action
+- [Scala 3] Scaladoc Completion for Scala3
+- [Scala 3] Completions in string interpolation
+- [Scala 2] Automatic import of types in string interpolations
+- More stable code navigation across subprojects
+- Code Action documentation
+- Support of Scala 3.2.0-RC3, Scala 3.2.0-RC2
+
+and a lot of bugfixes!
+
+## [Scala 3] Auto import and completion for extension methods
+
+You might know that Scala 3 has introduced `extension methods` that allow defining new methods to your existing types.
+
+Previously, Metals couldn't auto-complete extension methods; so developers had to find an appropriate extension method from their workspace and manually import it. But, this was time-consuming and not always beginner friendly.
+
+Now, Metals provides auto-completion for extension methods and automatically imports them!
+
+![extension-methods](https://i.imgur.com/EAbVHeH.gif)
+
+## [Scala 3] Convert to Named Parameters code action
+
+[Metals 0.11.7 added `ConvertToNamedParameters` code action to Scala2](https://scalameta.org/metals/blog/2022/07/04/aluminium#scala-2-add-converttonamedarguments-code-action).
+
+Thanks to the contribution by [@jkciesluk](https://github.com/jkciesluk), this feature is now available for Scala 3!
+
+![convert-to-named](https://i.imgur.com/9i7MWoQ.gif)
+
+## [Scala 3] Scaladoc completion
+
+Metals now supports offering Scaladoc completions in Scala 3. When typing `/**` you get an option to auto-complete a scaladoc template for methods, classes, etc.!
+
+![scala-doc-completion](https://i.imgur.com/MEJUXr3.gif)
+
+## [Scala 3] Completions in string interpolation
+
+In the previous versions, whenever users wanted to include a value in a string using string interpolation, they would need to do it all manually. Now, it is possible to get an automatic conversion to string interpolation when typing `$value`, as well as automatic wrapping in `{}` when accessing members of such value.
+
+![scala3-interpolation](https://i.imgur.com/EyFKpiv.gif)
+
+## [Scala 2] Automatically import types in string interpolations
+
+Previously, the only suggestions for string interpolations were coming from the currently available symbols in scope. This meant that if you wanted to import something from another package, you would need to do it manually.
+
+This problem is now resolved. Users can easily get such symbols automatically imported, which creates a seamless workflow.
+
+![scala2-inteprolation](https://i.imgur.com/cCWTQnj.gif)
+
+The feature also works for Scala 3.
```

Comment:
```suggestion
The feature also being worked on for Scala 3.
```
I haven't actually finished it :sweat:

Reply:
fixed 😄 https://github.com/scalameta/metals/pull/4214/commits/9620b840f2de62f10028678d1db7f6963e90f609